internal/command/command_test.go

```diff
@@ -0,0 +1,92 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var defaultDelims = []rune{'"', '\'', '`'}
+
+func TestGenerateFlagRegex(t *testing.T) {
+	specs := []struct {
+		name     string
+		flags    []string
+		expected string
+	}{
+		{
+			name:     "succeeds for single flag",
+			flags:    []string{"flag"},
+			expected: "flag"},
```

Comment:
Just a minor style thing, but I think this would be easier to read if the `}` were on the next line.